src/wrappers/ContentUploader.js

```diff
@@ -40,6 +40,16 @@ class ContentUploader extends ES6Wrapper {
         this.emit('error', data);
     };
 
+    /**
+     * Callback when a file uploads that gets called in the t3 page for Upload Embed Widget
+     *
+     * @param {Object} data - Upload item
+     * @return {void}
+     */
+    onStage = (data: BoxItem): void => {
```

Comment:
You didnt pass it to the component below

Reply:
Whoops, forgot to add that when I redid the PR